docs/troubleshoot/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Support and troubleshooting",
```

Comment:
Not sure why this didn't work... in the preview of the website, the label was not applied.

Reply:
I think I see it working? 
![image](https://github.com/deislabs/ratify-web/assets/12980641/9321824f-af3b-402a-95fa-a444ddf32591)

Reply:
For the `certstore` I think we need to update the title of the folder to be `Certificate Store`. For `installation-tips` we need to rename the title in the actual hackmd since that's what it's using I think.

Reply:
thanks! , i think i had some browser cache, i see it now too